index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Maher</title>
+    <link rel="stylesheet" href="css/style.css">
+    <!-- link swiper css -->
+    <link rel="stylesheet" href="CSS/swiper-bundle.min.css">
+    <!-- BOX ICONS -->
+</head>
+<body>
+    <!-- Header -->
+        <!-- nav -->
+        <div class="nav container">
+            <!-- Logo -->
+            <div class="navbar">
+            <p  href="#">Maher</p>
```

Comment:
- [ ] Kindly wrap your logo in an `<a>` tag instead of a `<p>` tag. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.This is for three reasons: convention, semantics, and user experience(UX).